### DISCUSSION TITLE
images/videos/etc. tab buttons don't work

### DISCUSSION THREAD

**Discussion by ghost** [searx/searx#2978] [Installation issue: Official wiki]
**Version of Searx, commit number if you are using on master branch and stipulate if you forked Searx**

commit `1ea573dc88ab37d05e47110d6080fda5b656cb81`
**How did you install Searx?**

i think by the wiki, using master branch
**What happened?**

when trying to switch tabs (i.e.  to images), nothing happens when clicking on them. Actually, they sort of act like checkboxes?

**How To Reproduce**

1. Go to a search
2. hit "images", "videos", or any of the other tabs

**Expected behavior**

It should switch to those tabs

**Screenshots & Logs**

https://user-images.githubusercontent.com/65276418/132153753-d8308135-c2b6-411b-aff0-90c56b67fa92.mp4

**Additional context**

not a browser issue, happened on librewolf as well

**Comment by kvch** [comment]
I cannot reproduce your issue. Also, the video you have attached seems corrupt. Could you please reload a video or give me more details about how to reproduce the problem?

**Reply by ghost** [reply]
This only affected [my instance](https://searx.swurl.xyz), but after a pull and reconfig, it works now, weird

**Comment by ghost** [comment]
pull + reconfig fixed it

**Comment by ghost** [comment]
The issue was that the "automatically reload when selecting a tab" option was off. Might have been defaulted in config

**Comment by fxbrit** [comment]
I'm having this issue whenever I un-comment the [lines to enable "Hostname replace" in settings.yml](https://github.com/searx/searx/blob/4d36aee57b99d6c72e5c96d05a76e6aa534ed664/searx/settings.yml#L110), 110-118.
